packages/react/src/patterns/sub-patterns/PictogramItem/README.md

```diff
@@ -0,0 +1,87 @@
+# Pictogram Item
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+@import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/pictogram-item';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import { Desktop } from '@carbon/pictograms-react';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { PictogramItem } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const heading = 'Lorem ipsum dolor sit amet.';
+  const copy = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.';
+
+  const cta = {
+    type: 'text',
```

Comment:
This should be defined inside the component once we get the CTA.

Reply:
I've moved the type to inside of the `PictogramItem.js`